voxel_buffer.h

```diff
@@ -86,7 +86,7 @@ class VoxelBuffer : public Reference {
 	void copy_from(const VoxelBuffer &other, unsigned int channel_index = 0);
 	void copy_from(const VoxelBuffer &other, Vector3i src_min, Vector3i src_max, Vector3i dst_min, unsigned int channel_index = 0);
 
-	_FORCE_INLINE_ bool validate_pos(unsigned int x, unsigned int y, unsigned int z) const {
+	_FORCE_INLINE_ bool validate_pos(int x, int y, int z) const {
 		return x < _size.x && y < _size.y && z < _size.z;
```

Comment:
_size was the one to cast instead

Reply:
Ok, I have cast each of the _size.x's to unsigned.